Week1/week-1 solution/solution3.py

```diff
@@ -0,0 +1,13 @@
+# [3]Write a python script to enter any number, if it is integer number, then check its palindrom or not. Print appropriate message if it is not palindrom.
+
+def palindrome(b):
+    a=str(b)

```

Comment:
explain the str()?

Reply:
str() is a built-in function that is used to convert different types of objects into a string representation.